README.md

```diff
@@ -48,6 +48,13 @@ sudo usermod -aG dialout <YOUR USERNAME>
 ```
 You will then need to log in and out again. Simply starting a new terminal is not sufficient. The Linux kernel will not refresh groups until the user completely logs out and logs in again.
 
+To use a PS4 controller you will need to install the driver and ros joy message. For that, do:
+
+```bash
+sudo pip install ds4drv
+sudo apt-get install ros-indigo-joy
```

Comment:
Can you add this `ros-indigo-joy` dependency to the `buzzmobile/package.xml`? Then I don't think we need to have it here because we can just install all deps declared there. All other deps like lidar and gps are there already.

Also, if `ds4drv` is a thing that's available as a rosdep (we'd have to look it up), then put it there too. Then we don't need this addition to the `README`.